Source/KSCrash/Recording/KSSystemCapabilities.h

```diff
@@ -38,7 +38,8 @@
 #define KSCRASH_HOST_IOS (KSCRASH_HOST_APPLE && TARGET_OS_IOS)
 #define KSCRASH_HOST_TV (KSCRASH_HOST_APPLE && TARGET_OS_TV)
 #define KSCRASH_HOST_WATCH (KSCRASH_HOST_APPLE && TARGET_OS_WATCH)
-#define KSCRASH_HOST_MAC (KSCRASH_HOST_APPLE && TARGET_OS_MAC && !(TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_WATCH))
+#define KSCRASH_HOST_VISION (KSCRASH_HOST_APPLE && TARGET_OS_VISION)
```

Comment:
I wonder what these checks are for? It wouldn't prevent compilation errors, as the preprocessor would complain that `TARGET_OS_VISION` is not defined. Only in case Android (or other OS) would have the same define?

Reply:
Do you mean it makes sense to check Xcode version before checking `TARGET_OS_VISION`? Yeah, in case there is no such define you would get a warning.

Reply:
Fixed this by checking if `TARGET_OS_VISION` is defined first